From: Christian Marillat <marillat@debian.org>
Date: Tue, 15 May 2012 04:49:36 -0300
Subject: Enable compilation on Debian arches other than Linux (kFreeBSD,
 HURD).
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

One of the main points here is that udev is present only on Linux (that is,
it is the exception, not the norm). So, we check if LINUX is defined or not
in the appropriate places, instead of listing every operating system that is
not Windows or MacOS X.

Applied-Upstream: https://github.com/HandBrake/HandBrake/commit/cd368d7e
Forwarded: yes
Last-Update: 2012-06-07

Reviewed-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Reviewed-by: Pino Toscano <pino@debian.org>
Signed-off-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
---
 gtk/configure.ac       |    9 +++++++--
 gtk/src/callbacks.c    |   11 ++++++++---
 libhb/ports.h          |    2 +-
 make/include/main.defs |    5 +++++
 4 files changed, 21 insertions(+), 6 deletions(-)

--- a/make/include/main.defs
+++ b/make/include/main.defs
@@ -69,6 +69,11 @@
     MODULES += gtk
 endif
 
+ifeq (1-gnu,$(FEATURE.gtk)-$(BUILD.system))
+    ## build gtk when gtk+gnu
+    MODULES += gtk
+endif
+
 ifeq (1,$(FEATURE.local_yasm))
     MODULES += contrib/yasm
 endif
